cmd/kube-apiserver/app/server.go

```diff
@@ -367,7 +366,7 @@ func BuildGenericConfig(s *options.ServerRunOptions) (*genericapiserver.Config,
 		return nil, nil, nil, nil, nil, err
 	}
 
-	genericConfig.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(generatedopenapi.GetOpenAPIDefinitions, api.Scheme)
+	genericConfig.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(openapi.GetOpenAPIDefinitions, api.Scheme)
```

Comment:
Running go test gave me `cmd/kube-apiserver/app/server.go:370: undefined: openapi.GetOpenAPIDefinitions`. 

But this makes sense, thanks. Closing.